tests/rundb/test_httpdb.py

```diff
@@ -593,9 +640,24 @@ def test_feature_sets(create_server):
         partition_order="desc",
     )
     assert len(feature_sets) == count, "bad list results - wrong number of members"
+    assert all([feature_set.status.stats for feature_set in feature_sets])
+    assert all([feature_set.status.preview for feature_set in feature_sets])
 
     feature_set = db.get_feature_set(name, project)
     assert len(feature_set.spec.features) == 4
+    # test minimal feature_sets list:
```

Comment:
It's better for comments to be written in English and not pseudo-code, but I don't think this is even a minimal feature set list? Minimal would be of size zero.
```suggestion
    # test minimal feature set list
```